rocky/rocky/templates/tasks/partials/tab_navigation.html

```diff
@@ -1,20 +1,22 @@
 {% load i18n %}
 
-<nav class="tabs" aria-label="{% translate "List of tasks" %}">
-    <ul>
-        <li {% if view == "boefjes_tasks" %}aria-current="page"{% endif %}>
-            {% if organization %}
-                <a href="{% url 'boefjes_task_list' organization.code %}">{% translate "Boefjes" %}</a>
-            {% else %}
-                <a href="{% url 'all_boefjes_task_list' %}">{% translate "Boefjes" %}</a>
-            {% endif %}
-        </li>
-        <li {% if view == "normalizers_tasks" %}aria-current="page"{% endif %}>
-            {% if organization %}
-                <a href="{% url 'normalizers_task_list' organization.code %}">{% translate "Normalizers" %}</a>
-            {% else %}
-                <a href="{% url 'all_normalizers_task_list' %}">{% translate "Normalizers" %}</a>
-            {% endif %}
-        </li>
-    </ul>
-</nav>
+<div class="hello-nav">
```

Comment:
Was this intentionally added, or just used for debugging?

Reply:
That was just for testing. I just removed the class, thanks!